### PR TITLE
Fixed issue 105 to work with metaboxes within custom post types.

### PR DESCRIPTION
**Pull request by filipecsweb**
This pull request fixes issue #105.

#### What's Included in this Pull Request
* My previous pull request fixed just the post type 'post'. Now it's working with CPTs. Sorry.
* I pulled all changes from master before changing this branch again. It's clean and well done.

### CODE REVIEW

File: adminimize.php [hunk 1]
```diff
@@ -870,7 +870,7 @@ function _mw_adminimize_set_metabox_cp_option() {
 
 	$user_roles                = _mw_adminimize_get_all_user_roles();
 	$_mw_adminimize_admin_head = '';
-	$metaboxes                 = '';
+	$metaboxes                 = [];
```
Comment (bueltge) [inline adminimize.php]: We need support for php5.2 :( so that we must use `array()` instead of `[]`. PLease can you change that. Thanks a lot.

File: adminimize.php [hunk 2]
```diff
@@ -879,22 +879,24 @@ function _mw_adminimize_set_metabox_cp_option() {
...
-				$metaboxes = implode( ',', $disabled_metaboxes_[ $current_post_type . '_' . $role ] );
+				$metaboxes[] = implode( ',', $disabled_metaboxes_[ $current_post_type . '_' . $role ] );
 			}
 		}
 	}
 
 	$_mw_adminimize_admin_head .= '<!-- Set Adminimize post options -->' . "\n";
 	$_mw_adminimize_admin_head .= '<style type="text/css">' .
-	                              $metaboxes . ' {display:none !important;}</style>' . "\n";
+	                              implode(',', $metaboxes) . ' {display:none !important;}</style>' . "\n";
```
Comment (bueltge) [inline adminimize.php]: Also here the plugin use the WP Codex, there need Whitespace after and before brackets, like `implode( ',', $metaboxes )`. Thx.

File: adminimize.php [hunk 3]
```diff
@@ -879,22 +879,24 @@ function _mw_adminimize_set_metabox_cp_option() {
 
 		if ( ! isset( $disabled_metaboxes_[ $current_post_type . '_' . $role ][ '0' ] ) ) {
 			$disabled_metaboxes_[ $current_post_type . '_' . $role ][ '0' ] = '';
+
+			continue;
 		}
 
-		$user = wp_get_current_user();
+		$user = $GLOBALS['current_user'];
```
Comment (bueltge) [inline adminimize.php]: Why this change? I think the function returns the current user, the ID so that usage of a globally is not necessary, or?

**Comment by filipecsweb** [comment]
Thanks for noticing those "mistakes". I've just pushed a new commit with them fixed.

Also, about `$GLOBALS['current_user']`, it's an instance of WP_User, just like `wp_get_current_user()` returns. So I'd say calling `$GLOBALS['current_user']` is a tiny bit faster.

**Comment by bueltge** [comment]
Thanks a lot @filipecsweb !
